aircraft/wing/structure.py

```diff
@@ -0,0 +1,84 @@
+from gpkit import Variable, Model
+
+class Structure(Model):
+    """
+    Structural model for a wing
+    source: Hoburg, Abbeel, "Geometric Programming for Aircraft Design Optimization", 2014
+    """
+
+    def setup(self, Lmax, fwadd=0.4, Nlift=2.0, rh=0.75, rhocap=2700, rhoweb=2700,
```

Comment:
I didn't think it made sense to have  a default value - the user should input the maximum load for the wing/tail being designed.